processing/src/main/java/org/apache/druid/query/scan/ScanQuery.java

```diff
@@ -334,7 +336,7 @@ public String toString()
            ", virtualColumns=" + getVirtualColumns() +
            ", resultFormat='" + resultFormat + '\'' +
            ", batchSize=" + batchSize +
-           ", limit=" + limit +
+           ", limit=" + scanRowsLimit +
```

Comment:
There are no backward compatibility concerns in `toString()`, please change the key.

Reply:
Changed